docs/developers/celestia-app-vesting.md

```diff
@@ -0,0 +1,404 @@
+---
+sidebar_label: How to create a vesting account with Celestia App
+description: Learn how to generate a vesting account using Celestia App.
+---
+
+# How to create a vesting account with Celestia App
+<!-- markdownlint-disable MD013 -->
+
+In this guide, we will learn how to create a vesting account using
+celestia-appd for both a local devnet and on Mocha testnet.
+
+## Local devnet
+
+First, download and [install Celestia App](../../nodes/celestia-app),
+selecting the
+[network and corresponding version](../../nodes/participate)
+that you would like to use.
+
+### Setting up the local devnet
+
+#### Run the devnet
+
+Next, change into the `$HOME/celestia-app` directory and
+run the single-node-devnet script.
+
+```bash
+cd $HOME/celestia-app
+./scripts/build-run-single-node.sh
+```
+
+#### Save the home directory path
+
+At the top of the output, you will see a path to the "Home directory",
+find yours from the output (**it will be unique every time**):
```

Comment:
[suggestion, optional] It would be helpful if we could add a sample output (we could conceal the output path though, so that users won't mistakenly use it)

Reply:
sample output like this? https://github.com/celestiaorg/docs/blob/bd09b29f19ce5d54a3337c74a8fea81c71c8b256/docs/developers/celestia-app-vesting.md?plain=1#L31-L47

how would you recommend concealing the path? i think having the full example makes it easier for someone _who is actually reading the guide_ to understand what type of path format to look for?

Reply:
> sample output like this?

Right! 👍 

> how would you recommend concealing the path? 

I think the current sample looks good.

Reply:
okay, thank you for the feedback! open to suggestions 🙏